Atlas/android/app/src/main/java/com/bounswe2017/group10/atlas/httpbody/CultureItem.java

```diff
@@ -66,6 +66,14 @@
     @Expose
     private boolean publicAccessibility;
 
+    @SerializedName("is_favorite")
+    @Expose
+    private boolean isFavorite;
+
+    @SerializedName("favorited_amount")
```

Comment:
Do the same steps for Parcelable implementation as you will do for ```isFavorite```.

Reply:
done @eozd